lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -433,7 +434,7 @@ def arrayCustomControlCreator(aeTemplate, c):
 def defaultControlCreator(aeTemplate, c):
     ufeAttr = aeTemplate.attrS.attribute(c)
     uiLabel = getNiceAttributeName(ufeAttr, c) if aeTemplate.useNiceName else c
-    cmds.editorTemplate(addControl=[c], label=uiLabel)
+    cmds.editorTemplate(addControl=[c], label=uiLabel, annotation=cleanAndFormatTooltip(ufeAttr.getDocumentation()))
```

Comment:
When debugging this I realized that we were only adding tooltips to the array and connected attributes. After discussion with Nat I added them to all the attributes.